Library/Homebrew/compilers.rb

```diff
@@ -1,7 +1,7 @@
 # @private
 module CompilerConstants
-  GNU_GCC_VERSIONS = %w[4.3 4.4 4.5 4.6 4.7 4.8 4.9 5]
-  GNU_GCC_REGEXP = /^gcc-(4\.[3-9]|5)$/
+  GNU_GCC_VERSIONS = %w[4.3 4.4 4.5 4.6 4.7 4.8 4.9 5 6]
```

Comment:
We could try to come up with some elaborate Ruby code, but for now this list is both sufficiently short and probably easier to read than any code we would replace it with.